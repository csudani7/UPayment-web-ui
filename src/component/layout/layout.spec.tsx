// #region Global Imports
import { render } from '@testing-library/react';
import * as React from 'react';
// #endregion Global Imports

// #region Local Imports
import Layout from './';

// #endregion Local Imports

describe('Basic Components', () => {
  describe('Layout', () => {
    it('should match snapshot', () => {
      const { container } = render(
        <Layout>
          <p> Hello </p>
        </Layout>,
      );
      expect(container).toMatchSnapshot();
    });
  });
});
