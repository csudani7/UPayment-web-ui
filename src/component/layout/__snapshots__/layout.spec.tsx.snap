// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic Components InputBase should match snapshot 1`] = `
<div>
  <div>
    <header
      class="w-full bg-indigo-600"
    >
      <nav
        aria-label="Top"
        class="px-4 mx-auto sm:px-6 xl:px-16"
      >
        <div
          class="flex items-center justify-between w-full py-6 border-b border-indigo-500 lg:border-none"
        >
          <div
            class="flex items-center cursor-pointer"
          >
            <p
              class="text-xl text-white"
            >
              UPayments Store
            </p>
          </div>
          <div
            class="ml-10 space-x-4"
          >
            <p
              class="text-xl text-white"
            >
              Register
            </p>
          </div>
        </div>
      </nav>
    </header>
    <p>
       Hello 
    </p>
  </div>
</div>
`;

exports[`Basic Components Layout should match snapshot 1`] = `
<div>
  <div>
    <header
      class="w-full bg-indigo-600"
    >
      <nav
        aria-label="Top"
        class="px-4 mx-auto sm:px-6 xl:px-16"
      >
        <div
          class="flex items-center justify-between w-full py-6 border-b border-indigo-500 lg:border-none"
        >
          <div
            class="flex items-center cursor-pointer"
          >
            <p
              class="text-xl text-white"
            >
              UPayments Store
            </p>
          </div>
          <div
            class="ml-10 space-x-4"
          >
            <p
              class="text-xl text-white"
            >
              Register
            </p>
          </div>
        </div>
      </nav>
    </header>
    <p>
       Hello 
    </p>
  </div>
</div>
`;
